tasks/audit-cat3.yml

```diff
@@ -1,6 +1,152 @@
-- name: "Place holder for Cat III Audits"
-  command: "true"
+---
+- name: "LOW | RHEL-07-010490 | PATCH | Unnecessary default system accounts must be removed."
+  shell: grep  {{ item }} /etc/passwd | wc -l
+  register: rhel_07_010490_audit
+  failed_when: no
+  changed_when: no
+  ignore_errors: yes
+  with_items:
+    - games
+    - ftp
+  tags:
+    - cat3
+    - low
+    - audit
+    - RHEL-07-010490
+
+- fail: msg="Unecessary default account present."
```

Comment:
Please do not use the Ansible shorthand syntax. Also, a `name` field is required per the contribution guidelines.

```
- name: "LOW | RHE-XX-XXXXXX | PATCH | Description"
  fail:
      msg: "Unnecessary default account present."
```